src/schema/tenantsGetter.ts

```diff
@@ -0,0 +1,36 @@
+import { isValidStringProperty } from '../validation/string';
+import Connection from '../connection';
+import { CommandBase } from '../validation/commandBase';
+import { Tenant } from '../openapi/types';
+
+export default class TenantsGetter extends CommandBase {
+  private className?: string;
+
+  constructor(client: Connection) {
+    super(client);
+  }
+
+  withClassName = (className: string) => {
```

Comment:
if this is required, doesn't it make more sense to have it directly in the constructor? (If that is possible)

Reply:
fixed